.circleci/config.yml

```diff
@@ -828,6 +828,22 @@ jobs:
           path: tmp/capybara
           destination: capybara
 
+  portafly:
+    resource_class: medium+
```

Comment:
Why medium+ do you really need that much of resource 😳  ? it costs $$$$

Btw I found another use of medium+https://github.com/3scale/porta/blob/d248f3221eb3d6dbeb302a433f63b4b80bf5373a/.circleci/config.yml#L506-L507

Can't we use `small`?

Reply:
https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Forig00.deviantart.net%2Fad8b%2Ff%2F2013%2F134%2Fc%2Fc%2Floadsamoney_gif_2_by_scruffy0105-d65973o.gif&f=1&nofb=1